src/views/docs/en/guides/developer-experience/using-esm.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Using ESM (ECMAScript modules)
+category: Developer experience
+description: How to use ECMAScript modules in functions
+---
+
+As of January 6, 2022 [AWS Lambdas support Node.js ES modules and top-level await](https://aws.amazon.com/blogs/compute/using-node-js-es-modules-and-top-level-await-in-aws-lambda/). Architect shipped an update in version `9.5.0` that adheres to Node.js conventions and the implementation from AWS.
+
+## CommonJS by default
+
+New and existing Architect projects will default to CommonJS. In the future `arc init` and `arc create` _might_ generate ESM functions. Join the discussion on [GitHub](https://github.com/architect/architect/discussions) and [Discord](https://discord.gg/y5A2eTsCRX) to help inform the future of Arc.
```

Comment:
I think we are definitely resolved to switch to ESM in Arc 10. Either way, I'd mention here that no matter what, as long as CJS is supported by Node.js and Lambda (which will be forever, heh), so too shall Architect. That is a hard commitment.